### PR TITLE
new readme, improved test suite, removed unnecessary test file

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
@jmburges 

addresses #6, #4, #3, #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,33 @@
...
 
-Hint 1. Don't assume your users are going to put in the commands exactly as you'd expect. It may be helpful to standardize your input before you compare it to account for all the possible scenarios of what a user might write in. [Ruby docs](http://ruby-doc.org/core-2.2.0/Array.html) -- there may already be a method that exists to clean up `gets` input.
-
-Hint 2. When creating your `play` method it may be helpful to draw out the different scenarios your app might encounter. Every if or elsif should result in a new branch of that tree. Then work your way back up by filling in pseudo-code for what needs to happen at each branch. 
-
-Hint 3. Proper indentation can be really helpful when you're dealing with complex if/else statements. It can be helpful to write a comment after each end to remind yourself which block or method it's closing. Using helper methods isn't required, but they can be helpful to break up your code so that it's easier to understand. 
+## Instructions
 
+* Take a look at `bin/jukebox`. This file is run by typing `ruby bin/jukebox`. Note that the file requires `lib/jukebox.rb`.
```
Comment (jmburges) [inline README.md]: I think we standarized on numbering instead of bullet points for instructions?

File: README.md [hunk 2]
```diff
@@ -1,49 +1,33 @@
...
-
-Hint 2. When creating your `play` method it may be helpful to draw out the different scenarios your app might encounter. Every if or elsif should result in a new branch of that tree. Then work your way back up by filling in pseudo-code for what needs to happen at each branch. 
-
-Hint 3. Proper indentation can be really helpful when you're dealing with complex if/else statements. It can be helpful to write a comment after each end to remind yourself which block or method it's closing. Using helper methods isn't required, but they can be helpful to break up your code so that it's easier to understand. 
+## Instructions
 
+* Take a look at `bin/jukebox`. This file is run by typing `ruby bin/jukebox`. Note that the file requires `lib/jukebox.rb`. 
+* Your `Jukebox` class should accept a list of songs upon initialization, and you need an `attr_accessor` for sogns. The `initialize` methods should set an `@songs` intance variable equal to the songs array that comes in as an argument.
```
Comment (jmburges) [inline README.md]: sogns

Comment (jmburges) [inline README.md]: intance

File: README.md [hunk 3]
```diff
@@ -1,49 +1,33 @@
...
+* `Jukebox` instances should respond to a `.call` method. This method is being invoked in the `bin` file, which, you might remember, is responsible for running the code in a command line app. The `.call` method should aggregate the other methods you will write and be resonsible for enacting the program. We'll come back to this method in a bit.
```
Comment (jmburges) [inline README.md]: the `bin` *folder*

Comment (jmburges) [inline README.md]: resonsible

Comment (jmburges) [inline README.md]: not enacting, executing

File: spec/01_jukebox_spec.rb [hunk 4]
```diff
@@ -70,6 +78,23 @@
     end
```
Comment (jmburges) [inline spec/01_jukebox_spec.rb]: Is there any way we can make the `prints the available commands` test a bit clearer? Those regexes may be hard for students to understand. Maybe the `include` matcher? All we really want to see if the help message includes `help`, `list`, `play` and `exit`.

File: spec/01_jukebox_spec.rb [hunk 5]
```diff
@@ -70,6 +78,23 @@
...
+      expect(jukebox.input).to eq("help")
+    end
+  end
+
+  describe '#do_command' do 
+    it 'calls the get_input method and invokes the appropriate method depending on the input of the user' do 
+      jukebox.stub(:gets).and_return("help")
+      get_input_output = jukebox.get_input
+      do_input_output = capture_stdout { jukebox.do_command }
+      expect(do_input_output).to match(/^(?=.*help)(?=.*list)(?=.*play)(?=.*exit).+/m)
```
Comment (jmburges) [inline spec/01_jukebox_spec.rb]: same comment as above. Any way we can do this without regex?

File: README.md (hunk 1 above)
Comment (SophieDeBenedetto) [inline README.md]: we are sticking w/ bullet points for now (i think is what we decided when we chatted on slack)

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

File: README.md (hunk 3 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed

File: spec/01_jukebox_spec.rb (hunk 4 above)
Comment (SophieDeBenedetto) [inline spec/01_jukebox_spec.rb]: fixed.

File: spec/01_jukebox_spec.rb (hunk 5 above)
Comment (SophieDeBenedetto) [inline spec/01_jukebox_spec.rb]: fixed.

**Comment by SophieDeBenedetto** [comment]
@jmburges, addressed all comments.

**Comment by AnnJohn** [comment]
@SophieDeBenedetto @jmburges Does this mean that it was merged (and so the issue should be closed) or that the issue needs to be closed? Just want to get some clarification on the workflow here

**Comment by SophieDeBenedetto** [comment]
@AnnJohn this means that i addressed all the comments joe raised on it and am waiting for joe to review my edits and then approve it/tell me to ship it.

**Comment by jmburges** [comment]
:shipit:

File: README.md (hunk 1 above)
Comment (jmburges) [inline README.md]: Yup!

**Pull request merged**